dotcom-rendering/src/web/layouts/FrontLayout.tsx

```diff
@@ -439,6 +470,33 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 					);
 				})}
 			</main>
+			<Section
+				fullWidth={true}
+				backgroundColour={palette.background.article}
+				showTopBorder={false}
+			>
+				<div css={trendingTopicContainer}>
+					<StraightLines
+						cssOverrides={css`
+							display: block;
+						`}
+						count={4}
+					/>
+					<div css={topicLabel}>Topics</div>
+					{/* TODO: Add allpath link */}
+					{front.trendingTopics?.map((tag) => {
+						return (
+							<a
+								key={tag.properties.webTitle}
+								href={tag.properties.webUrl}
+								css={linkStyle}
+							>
+								{tag.properties.webTitle}
+							</a>
+						);
+					})}
+				</div>
```

Comment:
I wonder if we'd want to break this into a `TrendingTopics` component, just so that the css/tsx can be self contained, and to avoid the FrontLayout being too large!